xbmc/windowing/X11/CMakeLists.txt

```diff
@@ -15,10 +13,8 @@ set(HEADERS GLContext.h
             OSScreenSaverX11.h
             WinEventsX11.h
             WinSystemX11.h
-            WinSystemX11GLContext.h
             XRandR.h
-            VideoSyncOML.h
-            X11DPMSSupport.h)
+            X11DPMSSupport.cpp)
```

Comment:
I guess the change from .h to .cpp for X11DPMSSupport was not on purpose as a cpp is not a header and the cpp is already added to SOURCES list some lines above.

Reply:
You're right, of course! See PR #16941.
I'm sorry for that.